app/Config/Schema/db_acl.php

```diff
@@ -12,91 +12,92 @@
-		'indexes' => array(
-			'PRIMARY' => array('column' => 'id', 'unique' => 1),
-			'idx_acos_lft_rght' => array('column' => array('lft', 'rght'), 'unique' => 0),
-			'idx_acos_alias' => array('column' => 'alias', 'unique' => 0)
-		)
-	);
-
-/**
- * ARO - Access Request Object - Something that wants something
- */
-	public $aros = array(
-		'id' => array('type' => 'integer', 'null' => false, 'default' => null, 'length' => 10, 'key' => 'primary'),
-		'parent_id' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
-		'model' => array('type' => 'string', 'null' => true),
-		'foreign_key' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
-		'alias' => array('type' => 'string', 'null' => true),
-		'lft' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
-		'rght' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
-		'indexes' => array(
-			'PRIMARY' => array('column' => 'id', 'unique' => 1),
-			'idx_aros_lft_rght' => array('column' => array('lft', 'rght'), 'unique' => 0),
-			'idx_aros_alias' => array('column' => 'alias', 'unique' => 0)
-		)
-	);
-
-/**
- * Used by the Cake::Model:Permission class.
- * Checks if the given $aro has access to action $action in $aco.
- */
-	public $aros_acos = array(
-		'id' => array('type' => 'integer', 'null' => false, 'default' => null, 'length' => 10, 'key' => 'primary'),
-		'aro_id' => array('type' => 'integer', 'null' => false, 'length' => 10, 'key' => 'index'),
-		'aco_id' => array('type' => 'integer', 'null' => false, 'length' => 10),
-		'_create' => array('type' => 'string', 'null' => false, 'default' => '0', 'length' => 2),
-		'_read' => array('type' => 'string', 'null' => false, 'default' => '0', 'length' => 2),
-		'_update' => array('type' => 'string', 'null' => false, 'default' => '0', 'length' => 2),
-		'_delete' => array('type' => 'string', 'null' => false, 'default' => '0', 'length' => 2),
-		'indexes' => array(
-			'PRIMARY' => array('column' => 'id', 'unique' => 1),
-			'ARO_ACO_KEY' => array('column' => array('aro_id', 'aco_id'), 'unique' => 1),
-			'idx_aco_id' => array('column' => 'aco_id', 'unique' => 0)
-		)
-	);
+    /**
+     * ARO - Access Request Object - Something that wants something.
+     */
+    public $aros = array(
+        'id' => array('type' => 'integer', 'null' => false, 'default' => null, 'length' => 10, 'key' => 'primary'),
+        'parent_id' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
+        'model' => array('type' => 'string', 'null' => true),
+        'foreign_key' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
+        'alias' => array('type' => 'string', 'null' => true),
+        'lft' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
+        'rght' => array('type' => 'integer', 'null' => true, 'default' => null, 'length' => 10),
```

Comment:
__[staging][code_sniffer]__
* Array key not aligned correctly; expected 20 spaces but found 8
* Array with multiple values cannot be declared on a single line